packages/lib/src/ref/core.ts

```diff
@@ -154,35 +182,61 @@ function getBackRefs(target: object, refType?: RefConstructor<object>): Observab
  *
  * @typeparam T Referenced object type.
  * @param target Node the references point to.
- * @param [refType] Pass it to filter by only references of a given type, or do not to get references of any type.
+ * @param refType Pass it to filter by only references of a given type, or do not to get references of any type.
+ * @param options Options.
  * @returns An observable set with all reference objects that point to the given object.
  */
 export function getRefsResolvingTo<T extends object>(
   target: T,
-  refType?: RefConstructor<T>
+  refType?: RefConstructor<T>,
+  options?: {
+    updateAllReferencesIfNeeded?: boolean
```

Comment:
Perhaps `updateAllRefsIfNeeded` to be more consistent with calling references "refs" everywhere else?

Reply:
Good point